NuKeeper.Integration.Tests/NuKeeper.Integration.Tests.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp2.0;net461</TargetFrameworks>
+    <TargetFrameworks>netcoreapp2.0</TargetFrameworks>
```

Comment:
You might want to change this to `TargetFramework` (no "s") as this is sort of multi-targeting but for 1 framework if you get me.

Reply:
Yep, that would explain why the VS tooling hadn't picked it up yet. An avoidable syntax complexity :(